src/main/java/training/subsystems/RobotArm/elbow/ElbowNEO.java

```diff
@@ -0,0 +1,58 @@
+package training.subsystems.RobotArm.elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ElbowNEO implements IElbow {
+
+	private final CANSparkMax motor;
+	private Rotation2d position;
+	private static ElbowNEO instance;
+
+	public ElbowNEO() {
+		this.motor = new CANSparkMax(ElbowConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		this.position = new Rotation2d(ElbowConstants.BIGINNING_POSITION);
+		motor.getPIDController().setP(ElbowConstants.KP);
+		motor.getPIDController().setI(ElbowConstants.KI);
+		motor.getPIDController().setD(ElbowConstants.KD);
+	}
+
+
+	public ElbowNEO getInstance() {
+		if (instance != null)
+			instance = new ElbowNEO();
+		return instance;
+	}
+
+
+	@Override
+	public void goToAngle(Rotation2d position) {
+		motor.getPIDController()
+			.setReference(
+				position.getDegrees(),
+				CANSparkBase.ControlType.kPosition,
+				ElbowConstants.PID_SLOT,
+				ElbowConstants.ARM_FEED_FORWARD.calculate(getPosition().getRadians(), getVelocity().getRotations())
```

Comment:
Radians or rotations? choose ONE method and stick with it throughout your entire code. In this case, it'll just cause weird FF consts, but it could cause problems in other cases.

Reply:
this is correct, the .calculate gets position radians and velocity